src/css/header.css

```diff
@@ -0,0 +1,39 @@
+#header {
+  display: -webkit-box;
+  display: flex;
+  height: 5em;
+  max-height: 5em;
+  -ms-flex-line-pack: center;
+  align-content: center;
+  padding: 2em 2em 6em 6em;
+}
+
+#header .logo {
+  height: 100%;
+  margin-right: 5em;
+  border: 2px solid black;
+  border-radius: 50%;
+  overflow: hidden;
+}
+
+#header .links {
+  display: -webkit-box;
+  display: flex;
+  height: 100%;
+}
+
+#header .links li {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  margin-right: 5em;
+}
```

Comment:
* You used flex on links that don't have items in them. I think removing display: flex is the right way to go